equinox/nn/conv.py

```diff
@@ -26,6 +26,52 @@ def parse(x: Any) -> tuple:
     return parse
 
 
+def compute_adjusted_padding(
+    input_size: int,
+    kernel_size: int,
+    stride: int,
+    padding: int,
+    output_padding: int,
+    dilation: int = 1,
+) -> Tuple[int, int]:
+    """Computes adjusted padding for desired ConvTranspose `output_padding`."""
+    output_size = (
+        (input_size - 1) * stride
+        - 2 * padding
+        + dilation * (kernel_size - 1)
+        + output_padding
+        + 1
+    )
+    kernel_size = (kernel_size - 1) * dilation + 1
```

Comment:
Nit: slightly neater to switch the definitions of `output_size` and `kernel_size`, as the latter is used in the former.